### DISCUSSION TITLE
Allow intent to play in radio mode

### DISCUSSION THREAD

**Discussion by markgaze** [music-assistant/support#2394] [Feature requests and ideas]
Currently, if you ask the voice assistant to play a track, it will only play that song. When asking Google Assistant to play a song, it will then move into radio mode and play other similar songs.

On my own instance, I've changed [this line](https://github.com/music-assistant/hass-music-assistant/blob/main/custom_components%2Fmass%2Fintent.py#L114) to `True` but it would be nice if this could be configured somewhere (I think there's another request for this to happen in Music Assistant generally) or if it was the default, I don't know if others expect it to play more songs after the requested track though!

**Comment by OzGav** [comment]
This idea of a smart queue is already on the backlog for a future release

**Reply by markgaze** [reply]
Yeah, I saw that one too, just figured this would be a simpler/quicker implementation (I'm happy to create a PR if we think that `True` is a better default in the intent 🙂 ) but if it's better to wait for the full implementation of the other request then I can just stick to changing it in my instance for now 🙂

**Reply by OzGav** [reply]
I don’t think we will be making a quick decision by changing that line as people may not want to enable radio mode all the time (for example, they just want to add a single song to an existing queue).

Having said that this request is about enabling radio mode by voice so that function can be separate to the smart queue concept. Will see what traction you get.

**Reply by markgaze** [reply]
Nice one, thanks!

**Reply by OzGav** [reply]
This has been implemented https://github.com/music-assistant/hass-music-assistant/pull/2913
